### ISSUE TITLE
sev-2: Veteran reporting issues with the provider picker list in secure messaging with VO

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#9442]
In the BVA / Section 508 monthly meeting, a screenreader user reported issues with the picker list for providers. I'm including the transcript below. 

I have an issue with the app
When I SM a doctor
Selecting the provider
Used to be you could swipe
It won’t let me now
Must swipe again to go back
It’s missing letters, it’s missing information
I could just swipe through the list
Sometimes the E is split up
I need the one in between
Having a tough time selecting the one I want
That’s a nuisance
I’m swiping and messing around with the list, driving me crazy
Like a month it’s been doing this

**Specs:** Veteran uses an iPhone / VoiceOver

**Comment by Sparowhawk** [comment]
might be tied to the solution for #9440

**Comment by brea11y** [comment]
That would be awesome, @Sparowhawk. It was a little difficult for us to understand what exactly was happening, but it sounded like with long lists of providers, it wasn't reading them all and skipping some? She kept mentioning that it was splitting them up. It sounded like she was having two issues. The first was navigating through a long list of providers in the picker and it skipping around or splitting them up oddly (she also mentioned it missing information?) and the second issue is that once she found the provider and selected it, she would have to navigate all the way back up through the options to get back to the "done" action to submit it. That's why I broke it up into two issues.

**Comment by brea11y** [comment]
I just ran through my husband's provider list with VoiceOver. He has 28 providers and uses an iPhone 15 (I know the Veteran who reported it is also on an iPhone) and I didn't notice any issues (other than understanding what some of these mean because they sound so odd with VO). It read out how I expected it to. I may reach out to the Veteran directly next week to try to get a better idea of what exactly she is experiencing.

**Comment by ala-yna** [comment]
@brea11y I'm going to move this to Pending Handoff for now while investigation continues.

**Comment by ala-yna** [comment]
Assigning to QA to see if the problem is reproduceable.

**Comment by ala-yna** [comment]
@DJUltraTom As the new QA lead around town, could you take a look at this ticket and decide if its replicable, and what the solution might be?

**Comment by DJUltraTom** [comment]
I have not been able to reproduce this issue using an ipad mini on iOS 17.7
issue could have been a result of an iOS bug that I believe @Sparowhawk had mentioned, I'll do another attempt on an iphone (tomorrow after it is charged up) and if we can't repro I'm good to close if there are no objections.

**Comment by DJUltraTom** [comment]
verified using iphone XS on ios 18.0.1 that I cannot reproduce this issue, closing as cannot repro
